html-css/override-extend-variables/step3-instructions.md

```diff
@@ -1,40 +1,42 @@
-## Extend a PatternFly component to achieve a new design
-
-Now that we have practiced overriding variables, let's practice extending PatternFly's system to create a new element in the card. Let's add a separator element to the card.
-
+Now that you have practiced overriding variables, you will practice extending PatternFly's system to create a new element in the card. You will add a separator element to the card.
+ 
 1) <strong>Add the html markup.</strong> 
 
-In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. We will prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.
-
+In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. You will prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.
```

Comment:
```suggestion
In `index.html` add the html markup for the separator element that is a sibling to the `pf-c-card__body` element. Add this new code between `pf-c-card__header` and `pf-c-card__body` and `pf-c-card__body` and `pf-c-card__footer`. Then prefix the variable name with `my-app` so that it can be easily distinguished from the PatternFly codebase.
```